handbook/engineering/security/index.md

```diff
@@ -15,21 +15,21 @@ We think that security is an enabler for the business. Sourcegraph is committed
+These goals represent our targeted work for 80% of our time. The remaining 20% is reserved for items that arise such as security reports. Our current work is documented in our [tracking issue](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+label%3Atracking+label%3Ateam%2Fsecurity+is%3Aopen). The work towards securing Sourcegraph Cloud is documented in our [productboard](https://sourcegraph.productboard.com/feature-board/2119755-cloud/features/5989081/detail).
 
 ### Visibility into Sourcegraph Cloud's attack surface
 
 **Problem:** The Cloud team is working toward [private code on Sourcegraph Cloud](../cloud/index.md#private-code-on-sourcegraph-cloud). Before we can have [confidence in our security model](#confidence-in-our-security-model), we first need to have visibility into current risks, issues, and threats, so we can proactively plan what work needs to be done (i.e., we don't want to be relying on or waiting for security researchers or customer to report vulnerabilities to us).
 
 **Milestones:**
 
-1. All Docker images are continuously scanned for known security vulnerabilities and the security team is alerted as vulnerabilities are found. (3.21 release)
-    1. Three or more vulnerabilities with a high severity, or a CVSS score of at least 4.0 are resolved. (3.21 release)
+1. All Docker images are continuously scanned for known security vulnerabilities and the security team is alerted as vulnerabilities are found. (3.22 release)
+1. Sourcegraph.com is behind a WAF and has had broad rate limits applied. (3.22 release)
 1. We have resolved container vulnerabilities with a high severity, or a CVSS score of at least 4.0, or that resolution is planned. (3.22 release)
-1. All compute nodes are continuously scanned for known security vulnerabilities and the security team is alerted as vulnerabilities are found. (3.22 release)
+1. All compute nodes are continuously scanned for known security vulnerabilities and the security team is alerted as vulnerabilities are found. (3.23 release)
     1. We have resolved compute vulnerabilities with a high severity, or a CVSS score of at least 4.0, or that resolution is planned. (3.23 release)
 1. Implement centralized storage of all of our existing logs (e.g., application logs, compute infrastructure logs). (3.23 release)
-1. Start collecting audit and access logs (e.g., visibility into both intentional and unintentional logins). This is useful from both a security visibility point of view, as also a requirement for various auditing frameworks (though not currently a target). (3.23 release)
+1. Start collecting audit and access logs (e.g., visibility into both intentional and unintentional logins). This is useful from both a security visibility point of view, as also a requirement for various auditing frameworks (though not currently a target). (3.24 release)
```

Comment:
Do we need this for GA, or can this be pushed off until just after? I can see arguments for either side.